src/components/Categories.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { getCategories } from "../api";
+import { Loading } from "./Loading";
+import { Link } from "react-router-dom";
+
+const Categories = () => {
+    const [isLoading, setLoading] = useState(false);
+    const [categories, setCategories] = useState([]);
+    useEffect(() => {
+        setLoading(true)
+        getCategories().then(({ categories }) => {
+            setLoading(false);
+            setCategories(categories);
+        })
+    }, []);
+    return isLoading ? <Loading /> : (
+        <ul className="categories">
+            {categories.map((category) => <li><Link to={`/categories/${category.slug}`}>{category.description}</Link></li>)}
```

Comment:
Also it might be a lot nicer for the user to see the category slug in the list because its more readable but I'm totally fine with you keeping it like this if this is what you've planned for the app